api/views.py

```diff
@@ -1,7 +1,42 @@
 from django.shortcuts import render
 
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+
+from lyrics.models import Lyric
+from lyrics.serializers import LyricSerializer
+
+from django.http import HttpResponse, JsonResponse
+
 
 def test(request):
     template = ""
     context = {}
     return render(request=request, template_name=template, context=context)
+
+
+@api_view(["GET"])
+def Get_LyricsList(request): # new 
+
+
+    try :
+        # first pick info from model
+        lyrics = Lyric.objects.all()
+    except Lyric.DoesNotExist:

```

Comment:
There was no need for this (I mean the `try` and `except`. Basically, when you do `Lyric.objects.all()`, a queryset is returned, This means when there are no lyrics no exception is raised but rather an empty set (or a set with no elements). So there's no need to wrap in a `try` and `except` block.